### ISSUE TITLE
C++ version

### ISSUE THREAD

**Issue by cduch9** [lxxue/FRNN#22]
Hi, this work is interesting. 

I would like to try and benchmark this algorithm from a C++ progam. The solution I have now is not performing to my satisfaction and yours might be better.  Do you have a recommandation on the way to do this performance test?

Thanks.

**Comment by lxxue** [comment]
Hi, I did the benchmark in python with `fvcore` [benchmark](https://github.com/facebookresearch/fvcore/blob/f3b07ea37daff5b774702708134f7b702e2dc83e/fvcore/common/benchmark.py#L76). An example of this can be found in this [script](https://github.com/lxxue/FRNN/blob/master/tests/bm_frnn_whole.py). I hope this helps.

**Comment by cduch9** [comment]
I want to try in C++ instead of Python.

**Comment by lxxue** [comment]
Then I guess you need to rewrite [frnn.py](https://github.com/lxxue/FRNN/blob/master/frnn/frnn.py) with C++ to have a C++ standalone implementation.

**Comment by cduch9** [comment]
Yes, and we must link with the C++ Distributions of PyTorch (https://pytorch.org/cppdocs/installing.html). I was kind of hoping that someone has done it before. 

I will stick to Python and convert my test data instead. You can close the issue if you want. Thanks.
